src/__mocks__/seleniumMock.js

```diff
@@ -0,0 +1,8 @@
+/* globals jest */
+/* eslint-disable no-unused-vars */
```

Comment:
I think you can apply this to the line and not the whole file,
there should be other examples in the project if you search for `ignore`

Reply:
updated